releases/v1.20.0

```diff
@@ -0,0 +1,44 @@
+# Highlights
+
+## Schedules API
+
+This version adds support for the schedule API to help users run workflows periodically.
+
+Note: the following two features will require support from the Temporal server, to be available in release v1.21.0.
+
+## Experimental: Worker Versioning
+
+This version adds support for our new Worker Versioning feature. Please note that this feature's API is subject to
+change. This feature allows you to assign build identifiers to workers, and ensure that workers operating on a task
+queue do not receive tasks that they will be incompatible with. You declare the relative [in]compatibility of build
+identifiers. We will be publishing documentation on this feature soon (and these notes will be updated with a link).
```

Comment:
Don't need the parens any more

Reply:
I think the docs link will not be stable so I will remove for now

Reply:
Hm, why not? Ought to be